.ci/magician/cmd/request_service_reviewers.go

```diff
@@ -0,0 +1,149 @@
+/*
+* Copyright 2023 Google LLC. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+package cmd
+
+import (
+	"fmt"
+	"magician/github"
+	"math/rand"
+	"os"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/magic-modules/tools/issue-labeler/labeler"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+// requestServiceReviewersCmd represents the requestServiceReviewers command
+var requestServiceReviewersCmd = &cobra.Command{
+	Use:   "request-service-reviewers PR_NUMBER",
+	Short: "Assigns reviewers based on the PR's service labels.",
+	Long: `This command requests (or re-requests) review based on the PR's service labels.
+
+	If a PR has more than 3 service labels, the command will not do anything.
+	`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		prNumber := args[0]
+		fmt.Println("PR Number: ", prNumber)
+
+		gh := github.NewClient()
+		execRequestServiceReviewers(prNumber, gh, labeler.EnrolledTeamsYaml)
+	},
+}
+
+func execRequestServiceReviewers(prNumber string, gh GithubClient, enrolledTeamsYaml []byte) {
```

Comment:
We probably want to have a short circuit exit condition for the terraform team. Changing generator level stuff might make noise for other teams (depending on the label application process).

Reply:
I don't think we want to exclude core contributors from this; large generator changes will be excluded by the ">3 teams impacted" condition.